packages/deploy/README.md

```diff
@@ -0,0 +1,131 @@
+# Platform Deployment Client
+
+Platform Deployment Client allows you to deploy contracts through the OpenZeppelin Platform, manage deployment configuration and manage block explorer api keys.
+
+## Install
+
+```bash
+npm install platform-deploy-client
+```
+
+```bash
+yarn add platform-deploy-client
+```
+
+## Usage
+
+Start by creating a new _Team API Key_ in Defender, and granting it the capability to manage sentinels. Use the newly created API key to initialize an instance of the Sentinel client.
+
+```js
+const { PlatformClient } = require('platform-deploy-client');
```

Comment:
Should we export separately DeploymentClient, DeploymentConfigClient and BlockExplorerClient?
This would mean that they should be instantiated on its own.

I think that users would mainly use DeploymentClient. DeploymentConfigClient and BlockExplorerClient would be used only during setup time.